src/components/specialized/Wallets/index.js

```diff
@@ -13,38 +13,85 @@ import Wallet from './Wallet'
 import withContainer from './container'
 import blockchains from 'configurables/blockchains'
 import network from 'configurables/network'
+import LinearProgress from '@material-ui/core/LinearProgress'
+import Button from '@material-ui/core/Button'
+import PaymentIcon from '@material-ui/icons/Payment'
+import PeopleIcon from '@material-ui/icons/People'
+import Login from '../Nav/Login'
+import Update from '../Achievements/Update'
+import Create from '../Achievements/Create'
```

Comment:
I prefer to use absolute imports:
`components/specialized/Achievements/Update` for ex
=> That way when moving files around twhen refactoring they still work